meta-iotqa/lib/oeqa/runtime/multimedia/audio/pulseaudio_rec_play.py

```diff
@@ -0,0 +1,28 @@
+import time
+from oeqa.oetest import oeRuntimeTest
+
+
+class PulseaudioTest(oeRuntimeTest):
+    def test_rec_play(self):
+        # Start pulseaudio daemon
+        (status, output) = self.target.run("pulseaudio -D")
+        self.assertEqual(status, 0, msg="Error pulseaudio not started: %s" % output)
+        # Recording audio
+        (status, output) = self.target.run("parecord -r /tmp/rec.wav &")
+        time.sleep(3)
+        self.assertEqual(status, 0, msg="Error not recorded: %s" % output)
+        # Stop pulseaudio daemon
+        (status, output) = self.target.run("pulseaudio -k")

```

Comment:
I know that parecord doesn't have time parameter to stop the recording, but maybe you could dig out or save the previously started parecord process id and kill thaq (instead of killing pulseaudio server)